x-pack/plugins/cases/public/components/visualizations/attachment.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import deepEqual from 'fast-deep-equal';
+import { LENS_ATTACHMENT_TYPE } from '../../../common/constants/visualizations';
+import * as i18n from './translations';
+
+import type {
+  PersistableStateAttachmentType,
+  PersistableStateAttachmentViewProps,
+} from '../../client/attachment_framework/types';
+import { AttachmentActionType } from '../../client/attachment_framework/types';
+import type { LensProps } from './types';
+import { OpenLensButton } from './open_lens_button';
+
+function getOpenLensButton(attachmentId: string, props: LensProps) {
```

Comment:
Lazy loading here would decrease the page bundle size iirc

Reply:
I think it is ok as we do not import any new component from other plugins and the `OpenLensButton` component is already in the bundle as it is used by the markdown editor.